script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_7.txt

```diff
@@ -117,10 +117,10 @@ It is as if she is declaring in advance that she won't listen to any of my compl
 "...Supervision, huh. So what would have happened if I had refused to come back to the Tohno house?"
 <Page34>
 "The freedom to decide that is not yours. No one from the Arima branch has the right to refuse a decision from the head family.
-You and I were both raised by Tohno household. At least until you reach adulthood and become an independen member of society, it is your obligation to serve the Tohno family."//　私も兄さんも、遠野家に育てられた者です。少なくとも成人し、社会的に自立するまでは、貴方は遠野家に隷属する義務があります」 first 遠野家 is "Tohno household" because I consider it a little ambiguous who raised him in this sentence.
+You and I were both raised by Tohno household. At least until you reach adulthood and become an independent member of society, it is your obligation to serve the Tohno family."//　私も兄さんも、遠野家に育てられた者です。少なくとも成人し、社会的に自立するまでは、貴方は遠野家に隷属する義務があります」 first 遠野家 is "Tohno household" because I consider it a little ambiguous who raised him in this sentence.
```

Comment:
this is already fixed in main by Waifwu's PR. funnily enough there's actually a second typo.

> You and I were both raised by Tohno household.